spec/jsSpecs/ares-install.spec.js

```diff
@@ -80,6 +80,7 @@ describe(aresCmd + ' --list(-l)', function() {
             if (stderr && stderr.length > 0) {
                 common.detectNodeMessage(stderr);
             }
+            expect(stdout).toContain(`Set target device to ${options.device}`);
```

Comment:
only ares-install is affected?
or, do you update tcs after fixed sentence?

Reply:
All effects to ares-commands which is to connect target. 

I put it representatively here. If you want to put it in all ares-commands that have an effect, I'll apply it to one TC on each command. I think the rest will be in common.

Reply:
I removed this logic.